ArduPlane/quadplane.cpp

```diff
@@ -495,7 +495,7 @@ const AP_Param::ConversionInfo q_conversion_table[] = {
     { Parameters::k_param_quadplane, 80,   AP_PARAM_FLOAT, "Q_P_ACCZ_I"},      //  Q_AZ_I
     { Parameters::k_param_quadplane, 144,  AP_PARAM_FLOAT, "Q_P_ACCZ_D"},      //  Q_AZ_D
     { Parameters::k_param_quadplane, 336,  AP_PARAM_FLOAT, "Q_P_ACCZ_IMAX"},   //  Q_AZ_IMAX
-    { Parameters::k_param_quadplane, 400,  AP_PARAM_FLOAT, "Q_P_ACCZ_FILT"},   //  Q_AZ_FILT
```

Comment:
we should check that @tridge is happy with this.  I think it is OK because those upgrading from a very old firmware (which used Q_AZ_FILT) will get the value copied over to the new parameter name (Q_P_ACCZ_FILTD) while those who were using a new firmware (Q_P_ACCZ_FILT) will be unaffected because the parameter is not moved in eeprom but rather just renamed.

I think the only people that will be caught out is those who have a stored parameter file somewhere.